cognite/client/data_classes/transformations/__init__.py

```diff
@@ -381,6 +400,33 @@ class TransformationList(CogniteResourceList):
     _RESOURCE = Transformation
 
 
+class TagsFilter(dict, CogniteFilter):
+    """Return only the resource matching the specified tags constraints.
+
+    Args:
+        contains_any (List[str]): The resource item contains at least one of the listed tags. The tags are defined by a list of external ids.
+        cognite_client (CogniteClient): The client to associate with this object.
+
+    Examples:
+
+            List only resources marked as a PUMP or as a VALVE::
+
+                >>> from cognite.client.data_classes import TagsFilter
+                >>> my_tag_filter = TagsFilter(contains_any=["PUMP", "VALVE"])
+    """
+
+    def __init__(self, contains_any: List[str] = None, cognite_client: "CogniteClient" = None):
+        self.contains_any = contains_any
+        self._cognite_client = cast("CogniteClient", cognite_client)
+
+    def dump(self, camel_case: bool = False) -> Dict[str, Any]:
+        dump_key = lambda key: key if not camel_case else utils._auxiliary.to_camel_case(key)
+        wrap = lambda values: None if values is None else [{"externalId": value} for value in values]
+        return {dump_key(key): wrap(value) for key, value in self.items()}
```

Comment:
don't understand this 😅 .... do we need to wrap in `{"externalId": _}`?

Reply:
it was copy paste, fixed it 👍